### DISCUSSION TITLE
Aspect Ratio Not Clickable / Not Usable?

### DISCUSSION THREAD

**Discussion by wipdesigns** [hlky/stable-diffusion-webui#1344] [Q&A] [answered]
Aspect Ratio Not Clickable / Not Usable when using google colab?

**Comment by codedealer** [comment, marked as answer]
Aspect ratio is just an info box, it's not meant to be interactive. Aspect ratio is controlled manually by adjusting width and height sliders.

**Reply by wipdesigns** [reply]
Thank you. I assumed it was like Midjourny's option to set the overall aspect ratio.
